developersociety/django-template
# SVG extra bits

Sources
None

Description
A couple of extra bits from the SVG work:
- I realised I'd put the sprite SVG files inside a folder called 'svg' within the 'sprite' folder due to the path specified in the webpack config, which was unnecessary - so have moved them back to just being directly in 'sprite' and have updated the webpack config urls accordingly. Most of the files in this PR are just moving those SVGs.
- The sprite.html template in 'includes' is no longer needed - this is now autogenerated and doesn't need to be touched by devs, and lives in /static/dist/svg.
- I added .DS_Store files to the gitignores as it annoys me every time I have to delete like 3 of them. 

Setup instructions
n/a

How to test
As the main functionality of this was already tested in #227 I think a look at the files is fine for this one, but you could also run through the instructions in #227  again if you really wanted to.

For comprehensive guidelines on code review at DEV, see here: https://www.notion.so/developersociety/Code-Review-b6d646b1d6e54011ada8169aee543231.
